kerastuner/engine/base_tuner.py

```diff
@@ -147,7 +147,7 @@ def run_trial(self, trial, *fit_args, **fit_kwargs):
         ```python
         def run_trial(self, trial, x, y, val_x, val_y):
             model = self.hypermodel.build(trial.hyperparameters)
-            model.fit(x, y)
```

Comment:
For our example, let's still use `model.fit`. We want to keep things as simple as possible in the user's minds

Reply:
Thanks for the comment. I'll make the change